### PR TITLE
Implement GraphQL mutation calls

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/network/graphDataStruct.js [hunk 1]
```diff
@@ -74,11 +60,253 @@ const FIELD_MAPPINGS = {
...
+        version
+        senderAddress
+        oracleAddress
+        optionIdx
+        amount
+      }
+    }
+  `,
+
+  approve: `
```
Comment (frankhu-optimizely) [inline src/network/graphDataStruct.js]: `approve` wont be in a separate mutation call. It is a separate transaction

Comment (frankobe) [inline src/network/graphDataStruct.js]: `approve` wont be in a separate mutation call, but it will be a separate transaction

Comment (dwalintukan) [reply src/network/graphDataStruct.js]: gotcha, removed

**Comment by dwalintukan** [comment]
@frankobe updated

**Pull request merged**